### PR TITLE
chore: Deploy snap and search indexer fixes to production.

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

**Comment by cafuego** [comment]
@lazysoundsystem I will tag and deploy this one (my) tomorrow.

File: config/user.role.anonymous.yml [hunk 1]
```diff
@@ -17,6 +16,5 @@ permissions:
   - 'access PDF version'
   - 'access comments'
   - 'access content'
-  - 'search content'
```
Comment (lazysoundsystem) [inline config/user.role.anonymous.yml:20]: Just checking this is intentional - and the changes in search permissions for admins and authenticated users too.

**Review by lazysoundsystem — APPROVED** [review]
Yup, as long as the permissions changes are intended.

File: config/user.role.anonymous.yml (hunk 1 above)
Comment (cafuego) [reply config/user.role.anonymous.yml:20]: Yes; these are core search permissions and not (as far as I know) used by search_api_db which is what GMS uses (though it uses the core search block as base for a custom block)

**Pull request merged**